tests/encodeinput.cpp

```diff
@@ -294,11 +300,12 @@ bool EncodeOutputVPX::init(const char* outputFileName, int width, int height, in
     return EncodeOutput::write(header, sizeof(header));
 }
 
-bool EncodeOutputVPX::write(void* data, int size)
+bool EncodeOutputVPX::write(void* data, int size, uint64_t timeStamp)
 {
     uint8_t header[12];
     memset(header, 0, sizeof(header));
     setUint32(header, size);
+    setUint64(&header[4], timeStamp);
```

Comment:
we can't write the timestamp directly to the frame header. It will generate a wrong bitstream if you use mp4 as input.
The ivf timestamp depends on two things
1. the time scale in the file header.
2. the time stamp in the frame header. 
we write the fps as time scale in the file header, The time stamp need be the frame number.

Reply:
Thanks for your details.
Is it okay if  I will replace timeStamp with frame number?